<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingMetricFastenerPitch</class>
 <widget class="QWidget" name="SettingMetricFastenerPitch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>135</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thread Pitch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix1/window/application_icon_.jpg</normaloff>:/new/prefix1/window/application_icon_.jpg</iconset>
  </property>
  <widget class="QWidget" name="background_" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>135</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QLabel" name="upper_band_">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>135</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ok_">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>175</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="close_">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>5</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name_widget_">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Шаг метр. резьбы</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
